lblod/ember-rdfa-editor-lblod-plugins
# Update to latest version of editor and fix issues with typescript

### Overview
This PR contains the following changes:
- Update the plugins where needed to be conform with latest version of https://github.com/lblod/ember-rdfa-editor/pull/1021
- Volta: pin node version to 20 (same as the one on the editor)
- Fix some import issues


##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor/pull/1021


### Setup
- Checkout latest version of https://github.com/lblod/ember-rdfa-editor/pull/1021
- `yalc publish`
- `yalc add @lblod/ember-rdfa-editor`

### How to test/reproduce
- Run `npm run prepack` and verify that no type issues occur
- Run the application and check for any runtime issues. Note: this PR should not contain any feature changes.

### Challenges/uncertainties
Due to the change from `ember-cli-typescript` to simply using `tsc`, `npm link` is not yet fully working. (there are still several unexplained type issues). This issue does not occur with `yalc` or `npm pack`.

### Checks PR readiness
